bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_SMEFTsim/WWTolnulnu_LO/WWTolnulnu_LO_run_card.dat

```diff
@@ -0,0 +1,278 @@
+  0.0    =  ptl1min  ! minimum pt for the leading lepton in pt
+  0.0    =  ptl2min  ! minimum pt for the second lepton in pt
+  0.0    =  ptl3min  ! minimum pt for the third lepton in pt
+  0.0    =  ptl4min  ! minimum pt for the fourth lepton in pt
+  -1.0    =  ptl1max  ! maximum pt for the leading lepton in pt
+  -1.0    =  ptl2max  ! maximum pt for the second lepton in pt
+  -1.0    =  ptl3max  ! maximum pt for the third lepton in pt
+  -1.0    =  ptl4max  ! maximum pt for the fourth lepton in pt
+#*********************************************************************
+# Control the Ht(k)=Sum of k leading jets                            *
+#*********************************************************************
+  0.0    =  htjmin  ! minimum jet HT=Sum(jet pt)
+  -1.0    =  htjmax  ! maximum jet HT=Sum(jet pt)
+  0.0    =  ihtmin   !inclusive Ht for all partons (including b)
+  -1.0    =  ihtmax   !inclusive Ht for all partons (including b)
+  0.0    =  ht2min  ! minimum Ht for the two leading jets
+  0.0    =  ht3min  ! minimum Ht for the three leading jets
+  0.0    =  ht4min  ! minimum Ht for the four leading jets
+  -1.0    =  ht2max  ! maximum Ht for the two leading jets
+  -1.0    =  ht3max  ! maximum Ht for the three leading jets
+  -1.0    =  ht4max  ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+  0.0    =  ptgmin  ! Min photon transverse momentum
+  0.4    =  R0gamma  ! Radius of isolation code
+  1.0    =  xn  ! n parameter of eq.(3.4) in hep-ph/9801442
+  1.0    =  epsgamma  ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+  True    =  isoEM  ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+  0.0    =  xetamin  ! minimum rapidity for two jets in the WBF case
+  0.0    =  deltaeta  ! minimum rapidity for two jets in the WBF case
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+  -1.0    =   ktdurham
+   0.4    =   dparameter
+ #*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+  5    =  maxjetflavor     ! Maximum jet pdg code
```

Comment:
This is not consistent with your proc_card.

define p = g u c d s u~ c~ d~ s~
define j = g u c d s u~ c~ d~ s~

Can you check with your model file whether b mass is set to 0 or not?

Reply:
b is indeed massive. I have fixed it